content/05.Experimental_Design.md

```diff
@@ -2,14 +2,19 @@
 
 <!-- Leader: Will Lewis; co-leaders: James Martin -->
 
-## Repetition 
+## Randomization 
+This section will discuss the importance of randomly assigning treatments to groups in experimental studies. The treatment is not ranomly applied to groups in observational studies, though methods such a random selection of study sites and survey areas can help to reduce bias. Some of this may overlap with previous sections describing the difference between experimental and observational studies.
 
 ## Replication 
+This section will discuss the importance of indepndent replicates for experimental and observation studies, including examples to show how incorrect inferences may be generated with insufficient levels of replication. Observational studies are correlative by nature, so inferences are generally weak without sufficient levels of replication. How to determine the experimental unit of analysis and avoiding the pitfalls of pseudoreplication will be discussed. This section will also discuss the value of a pilot study and performing a prospective power analysis for estimating the necessary level of replication within groups to observe a desired effect size.
 
-## Randomization 
+## Repetition 
+This section will discuss the use of repeated measuresurements within experimental units, for the purposes of estimating precision or more accuratley measuring the response variable. Methods incorporating repetition, such as the nested design, will be discussed. This section will highlight the difference between replication and repitition and the need to distinguish between the two to avoid pseudoreplication.
 
 ## Controls 
+This section will discuss the importance of including a control treatment in experimental studies, including an example showing how incorrect inferences may be generated without an adequate control. This section will also discuss appropriate methods of designing a control so that differences between treatment and control groups may be attributed to the treatment. For observational studies, it is important that replicates differ in their level of the explanatory variable in order to draw inferences about the effect on the response variable. 
 
-## Blocking 
+## Blocking and Controlling Variation
+This section will discuss methods of accounting for extraneous variation, giving examples of how incorrect inferences may be generated by not accounting for extraneous variation. This section will discuss the use of methods of accounting for variation in experimental studies, such as blocking or restriction. When possible, study areas should be interspersed to avoid bias from environmental gradients in confounding variables. Observational studies frequently rely on covariates to account for extraneous variation, though proper experimental design is still important.
```

Comment:
Make sure to include a discussion about bad controls and bad covariates or adjustment variables.